integrationTests/vm/testInitializer.go

```diff
@@ -753,6 +756,15 @@ func createDefaultVMConfig() *config.VirtualMachineConfig {
 	}
 }
 
+type ResultsCreateTxProcessor struct {
```

Comment:
add mock comment? also, please rename to `CreateTxProcessorResults`

Reply:
done